src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -91,23 +88,46 @@ public class ExtensionsRunner {
     private static final String NODE_NAME_SETTING = "node.name";
 
     private String uniqueId;
-    private DiscoveryNode opensearchNode;
+    /**
+     * This method is call from {@link ExtensionsInitRequestHandler}.
```

Comment:
Not a method. To better frame it like: `Initializes OpenSearch Node`

Reply:
Well, the field doesn't initialize, it gets initialized.  So I think a mix of what the two of you have here.  "This field is initialized by a call from ...."